tensorflow/cc/framework/cc_ops_test.cc

```diff
@@ -89,9 +89,9 @@ TEST(CCOpTest, MultiOutput) {
   auto u = Unique(root, {1, 2, 2, 4, 3, 2});
   std::vector<Tensor> outputs;
   test::GetTensors(root, {u.y, u.idx}, &outputs);
-  test::ExpectTensorEqual<int>(outputs[0], test::AsTensor<int>({1, 2, 4, 3}));
+  test::ExpectTensorEqual<int>(outputs[0], test::AsTensor<int>({1, 2, 3, 4}));
   test::ExpectTensorEqual<int>(outputs[1],
-                               test::AsTensor<int>({0, 1, 1, 2, 3, 1}));
+                               test::AsTensor<int>({0, 1, 1, 3, 2, 1}));
```

Comment:
confirm testcase pass in CPU & GPU mode

Reply:
Already confirmed that this unit test runs in GPU mode if possible.